src/index.html

```diff
@@ -1,17 +1,169 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label class="form-field">
+            Name:
+            <input
+              type="text"
+              name="firstName"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label class="form-field">
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input type="date"name="dateOfBirth">
```

Comment:
```suggestion
            <input type="date" name="dateOfBirth">
```
Don't forget about spaces between attributes.